### PR TITLE
Made installable with PHPUnit 6.x

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Fixes https://github.com/oradwell/covers-validator/issues/11

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
A fast release would be cool; this is very annoying :)

**Comment by JeroenDeDauw** [comment]
Derp... I ran with the wrong version of PHPUnit and missed a "few" things :) Will submit a new version that requires PHP 7 (too hard to support all)

**Comment by JeroenDeDauw** [comment]
> There was 1 failure:

>1) OckCyp\CoversValidator\Tests\Loader\FileLoaderTest::testLoadsFile2
Failed asserting that true is false.

> .../covers-validator/tests/Loader/FileLoaderTest.php:26

I don't understand why this is failing and the test before does not, or what exactly those tests do...

**Comment by oradwell** [comment]
Thanks a lot for this! I'll have a look soon.

File: tests/Loader/TestSuiteLoaderTest.php [hunk 1]
```diff
@@ -12,11 +12,11 @@ class TestSuiteLoaderTest extends BaseTestCase
      */
     public function testLoadsSuite()
     {
-        $testSuite = $this->getMockBuilder('PHPUnit_Framework_TestSuite')
+        $testSuite = $this->getMockBuilder('PHPUnit\Framework\TestSuite')
             ->disableOriginalConstructor()
             ->getMock();
 
-        $configuration = $this->getMockBuilder('PHPUnit_Util_Configuration')
+        $configuration = $this->getMockBuilder('PHPUnit\Util\Configuration')
```
Comment (gbirke) [inline tests/Loader/TestSuiteLoaderTest.php]: Why not use `PHPUnit\Util\Configuration::class` ?

Comment (JeroenDeDauw) [inline tests/Loader/TestSuiteLoaderTest.php]: I did not get to doing that everywhere. Step one would be to get the tests to work again :)

**Comment by oradwell** [comment]
I am having a look now.

**Comment by oradwell** [comment]
I did the necessary fixes here: https://github.com/oradwell/covers-validator/tree/JeroenDeDauw-phpunit6x

not sure why PHP globals are no longer removed between the tests.
My worry is coverage going down. For some reason the command is not finding any test methods.

**Pull request merged**